instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/TracingExecutionInterceptor.java

```diff
@@ -53,9 +50,9 @@ public void afterMarshalling(
     Span span = executionAttributes.getAttribute(SPAN_ATTRIBUTE);
 
     if (span != null) {
-      DECORATE.onRequest(span, context.httpRequest());
-      DECORATE.onSdkRequest(span, context.request());
-      DECORATE.onAttributes(span, executionAttributes);
+      TRACER.onRequest(span, context.httpRequest());
+      TRACER.onSdkRequest(span, context.request());
+      TRACER.onAttributes(span, executionAttributes);
```

Comment:
i think consolidating these into a single method would be nice, e.g.
```suggestion
      TRACER.afterMarshalling(span, context, executionAttributes);
```

Reply:
Create a method called afterMarshalling in AwsSdkClientTracer or HttpClientTracer? There is no such a method in TRACER yet.